src/parser/lval.js

```diff
@@ -124,6 +110,28 @@ export default class LValParser extends NodeUtils {
     return node;
   }
 
+  parseAssignableObjectExpressionProp(
```

Comment:
Can this be renamed to `toAssignableObjectExpressionProp`, since it checks and converts an _already parsed_ property?